core/errcheck/errcheck.go

```diff
@@ -9,11 +9,15 @@ import (
 // CheckCases checks error cases for tests
```

Comment:
I would revert this file -- these are just line-length changes.

Reply:
Here's a SO article that might help: https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision